Support block comments

jpcima opened the pull request:
#195 

This adds support of block comments in the parser.
There are some examples in the parser demo.

Maybe not all possibilities are supported.
For example, don't expect to be able to write: `sample=/*blabla*/mycomment.wav`

If the comment is not terminated with `*/`, an error will be raised.

jpcima committed:
Support block comments

jpcima committed:
Add some parsing tests

paulfd commented on src/sfizz/parser/Parser.h:
```diff
@@ -84,8 +84,8 @@ class Parser {
     // helpers
-    static bool hasComment(Reader& reader);
-    static size_t skipComment(Reader& reader);
+    static int hasComment(Reader& reader); // 0=None 1=Line 2=Block
```
Maybe an enum would be slightly clearer here for the future

jpcima committed:
Change the comment type into an enum

jpcima committed:
Make the skip count correct (even if not used currently)

jpcima commented:
- changed comment type to an enum
- modified `skipComment` so the character count returned is correct
  (note it doesn't prevent the parser to work correctly, but better to fix it regardless)
